app/src/main/java/org/simple/clinic/summary/assignedfacility/AssignedFacilityUiRenderer.kt

```diff
@@ -7,6 +7,9 @@ class AssignedFacilityUiRenderer(
 ) : ViewRenderer<AssignedFacilityModel> {
 
   override fun render(model: AssignedFacilityModel) {
-
+    if (model.hasAssignedFacility) {
+      ui.showAssignedFacility()
+      ui.renderAssignedFacilityName(model.assignedFacility!!.name)
```

Comment:
When I see these two together, I don't know why they are different since I would assumed rendering the name of the assigned facility will be handled internally by the `showAssignedFacility` method. Can we rename the methods to better reflect why they are different?

Reply:
This is also why I think the responsibility of hiding/showing the view itself should be the responsibility of the parent view and not this view.

Reply:
Yeah, I will move the view controls to parent view